games/minimal/mods/default/init.lua

```diff
@@ -1270,7 +1270,9 @@ minetest.register_node("default:chest_locked", {
 	sounds = default.node_sound_wood_defaults(),
 	after_place_node = function(pos, placer)
 		local meta = minetest.get_meta(pos)
-		meta:set_string("owner", placer:get_player_name() or "")
+		local pname =
+			placer and placer:is_player() and placer:get_player_name() or ""
```

Comment:
Redundant `is_player` check. The `get_player_name` function returns an empty string if it's no player.

Reply:
Thanks, didn't know.